unstructured/partition/csv.py

```diff
@@ -31,8 +31,6 @@ def partition_csv(
         A string defining the target filename path.
     file
         A file-like object using "rb" mode --> open(filename, "rb").
-    metadata_filename
```

Comment:
Did you mean to remove this section from the doc strings?

Reply:
Oh yeah I forgot that was included in the docstring from the decorator now. Good to go

Reply:
So, that means I should remove `metadata_filename` from the docstring and let it be added via the decorator, but leave the order of the parameters for the functions unchanged (meaning it's okay of the order of the parameters isn't the same as the order in the docstring), right?